root/static/less/print.less

```diff
@@ -1,4 +1,45 @@
 @media print {
     .pod { margin: 0px; border: 0px solid #000; padding: 0px}
     .home, .header, .search-bar, .box-right, .footer, a.favorite {display:none}
+    #page-pod {
+        .sticky-panel-top,
+        .sticky-panel-sticky,
+        .dependencies,
+        .search-form,
+        .head-small-logo,
+        .dropdown,
+        #libwww-perl-fav,
```

Comment:
This selector still doesn't make any sense.

Reply:
My bad, I've changed that ID to a class .logged_in